src/backend/wine/manager/utils.ts

```diff
@@ -104,6 +106,24 @@ function getInstallDir(release: WineVersionInfo): string {
   } else if (release.type.includes('Toolkit')) {
     return `${toolsPath}/game-porting-toolkit`
   } else {
+    // Check if we want to download Proton-GE directly to Steam
+    if (release.type === 'GE-Proton') {
```

Comment:
`release.type` can only be `GE-Proton` or `Proton` here (not sure how `Proton` can happen in the first place, did we ever list normal Proton in the Wine Manager?), meaning this `if` can probably be removed

Reply:
done